src/bin/pg_autoctl/nodestate_utils.c

```diff
@@ -382,6 +391,92 @@ nodestateHealthToString(int health)
 }
 
 
+/*
+ * Transform the health column from a monitor into a single char.
+ */
+char
+nodestateHealthToChar(int health)
+{
+	switch (health)
+	{
+		case -1:
+		{
+			return '?';
+		}
+
+		case 0:
+		{
+			return '!';
+		}
+
+		case 1:
+		{
+			return '*';
```

Comment:
```suggestion
			return '✓';
```

Reply:
Maybe this being a space as also totally fine, i.e. no need to have a character shown if everything is good.

Reply:
I don't know how to maintain table alignment when using wider unicode chars, and unfortunately both ✓ and ✗ are wider than space.